roles/installer/ospd/overcloud/scale/tasks/main.yaml

```diff
@@ -0,0 +1,36 @@
+- name: Set scaler variables
+  set_fact:
+    scale_service={{ item.key }}
+    scale_amount={{ item.value }}
+  when: (item.key == "ceph-storage") or
```

Comment:
yes,I was trying to implement logic from this comment :
- {role: installer/ospd/overcloud/scale, compute=1} 
- {role: installer/ospd/overcloud/scale, compute=2} 
and parse variable name 'compute' from hostvars, and "when" is not needed when call role like that:
- {role: installer/ospd/overcloud/scale, scale_service:  'compute',  scale_amount: 1}